Sources/JobsPostgres/PostgresJobsQueue.swift

```diff
@@ -44,6 +44,10 @@ import PostgresNIO
 /// ```
 public final class PostgresJobQueue: JobQueueDriver {
     public typealias JobID = UUID
+    public struct JobData {
```

Comment:
I added this structure to so that getJobs can return job_id and queued_at

Reply:
I'm wondering if it best to just use the queued time from `_hb_pg_jobs` and not bother with the value in `_hb_pg_job_queue`.

You can get the queuedAt at the same time you get the job data
`SELECT job, queued_at FROM _hb_pg_jobs WHERE id = \(jobId) FOR UPDATE SKIP LOCKED`.

Reply:
I can drop the column `queued_at` from the `_hb_pg_job_queue`, it does not serve any purpose right now. I could return a tuple (JobID, Date) from the getJobs instead of `JobData`